.github/workflows/deploy.yml

```diff
@@ -0,0 +1,99 @@
+name: Deploy
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    tags:
+      # pre-release tag
+      - "202[3-9].[0-9][0-9].[0-9]+-rc[0-9]+"
+      # release tags
+      - "202[3-9].[0-9][0-9].[0-9]+"
```

Comment:
Nitpick but we could start these in 2024, we won't be making a 2023 release :grin: